components/vocabularies.jsx

```diff
@@ -0,0 +1,187 @@
+  },
+  {
+    id: uuidv4(),
+    word: "Teller",
+    article: "Der",
+    wordType: "Noun",
+    ipa: "ˈtɛlər",
+    category: "food",
+    rating: 10,
+    translation: "Plate",
+  },
+  {
+    id: uuidv4(),
+    word: "Essen",
+    article: "Das",
+    wordType: "Noun",
+    ipa: "ˈɛsən",
+    category: "food",
+    rating: 10,
+    translation: "Food",
+  },
+  {
+    id: uuidv4(),
+    word: "Küchenutensil",
+    article: "Das",
+    wordType: "Noun",
+    ipa: "ˈkʏçn̩ʔutɛnˌziːl",
+    category: "food",
+    rating: 30,
+    translation: "kitchen utensil",
+  },
+  {
+    id: uuidv4(),
+    word: "Wunderbar",
+    article: "",
+    wordType: "Adjective",
+    ipa: "ˈvʊndɐˌbaːɐ̯",
+    category: "Adjective",
+    rating: 20,
+    translation: "Marvelous",
+  },
+  {
+    id: uuidv4(),
+    word: "Ängstlich",
+    article: "",
+    wordType: "Adjective",
+    ipa: "ˈɛŋstlɪç",
+    category: "Adjective",
+    rating: 20,
+    translation: "",
+  },
+  {
+    id: uuidv4(),
+    word: "Ängstlich",
+    article: "",
+    wordType: "Adjective",
+    ipa: "ˈɛŋstlɪç",
+    category: "Adjective",
+    rating: 20,
+    translation: "Scared",
+  },
+
+  {
+    id: uuidv4(),
+    word: "Chronisch",
+    article: "",
+    wordType: "Adjective",
+    ipa: "ˈkʁoːnɪʃ",
+    category: "Adjective",
+    rating: 20,
+    translation: "Chronicly",
+  },
+
+  {
+    id: uuidv4(),
+    word: "Ausgefuchst",
+    article: "",
+    wordType: "Adjective",
+    ipa: "ˈaʊ̯sɡəˌfʊkst",
+    category: "Adjective",
+    rating: 30,
+    translation: "Clever",
+  },
+  {
+    id: uuidv4(),
+    word: "Bus",
+    article: "Der",
+    wordType: "Noun",
+    ipa: "bʊs",
+    category: "Transportation",
+    rating: 10,
+    translation: "Bus",
+  },
+  {
+    id: uuidv4(),
+    word: "Bushaltestelle",
+    article: "Die",
+    wordType: "Noun",
+    ipa: "ˈbʊshaltəˌʃtɛlə",
+    category: "Transportation",
+    rating: 30,
+    translation: "Bus stop",
+  },
+  {
+    id: uuidv4(),
+    word: "Fliegen",
+    article: "Das",
+    wordType: "Substantiv",
+    ipa: "ˈfliːɡn̩",
+    category: "Transportation",
+    rating: 20,
+    translation: "Flying",
+  },
+  {
+    id: uuidv4(),
+    word: "bogenschießen",
+    article: "",
+    wordType: "Substantiv",
+    ipa: "ˈboːɡn̩ˌʃiːsn̩",
+    category: "Sport",
+    rating: 30,
+    translation: "Archery",
+  },
+  {
+    id: uuidv4(),
+    word: "programmieren",
+    article: "",
+    wordType: "Verb",
+    ipa: "pʁoɡʁaˈmiːʁən",
+    category: "Beruf",
+    rating: 30,
+    translation: "To Program",
+  },
+  {
+    id: uuidv4(),
+    word: "Henne",
+    article: "Die",
+    wordType: "Noun",
+    ipa: "-ɛnə",
+    category: "Tier",
+    rating: 10,
+    translation: "Hen",
+  },
+  {
+    id: uuidv4(),
+    word: "merkwürdig",
+    article: "",
+    wordType: "Adjektiv",
+    ipa: "ˈmɛʁkˌvʏʁdɪç]",
+    category: "Adjektive",
+    rating: 30,
+    translation: "Remarkable",
+  },
+];
+export default vocabularies;
```

Comment:
You are using a `.jsx` file extension. That is only nessecary when using JSX code. Use `.js` instead.

This file also seems to be your "database". While I am ambivalent about whether it's better to use `json` vs. `js` here, I'd advise to not use uuids here.

Every time your app is rendered by the server or client these `id`s will be different giving you a mismatched render between server and client, as well as making your ids very unstable (e.g. unusable for the url, because they keep changing).

Create your `id`s by hand and store them hardcoded. Alternatively you could use a real database.

Reply:
Thanks! I implemented your suggestions and I'll probably use a database at a later stage.